### ISSUE TITLE
Fluid field incorrect rendering in preview

### ISSUE THREAD

**Issue by acutedeveloper** [ExpressionEngine/ExpressionEngine#4388]
**Description of the problem**
When using Fluid field groups in an entry. The preview is not rendering correctly. I had initially being using [coilpack and blade templates](https://github.com/ExpressionEngine/Coilpack/issues/12), but thought I would check again if this was a native EE thing.

If I set a group to represent a component like a hero banner, in the preview it has the appearance that the component has been output multiple times. Each time for each field in the field group.

On the front end the fluid field group outputs as expected.

I would rank this as **critical** as clients will be expected to use the preview feature.

**How To Reproduce**
This is my fluid field with a field group

<img width="813" alt="Screenshot 2024-07-10 at 16 46 57" src="https://github.com/ExpressionEngine/Coilpack/assets/2130276/7e1fab4d-1fe5-4bce-af10-8f10f37bbb2d">

This is my template.

```
{exp:channel:entries channel="pages"}
    {title}
    {ff_webpage_layout}
        {ff_webpage_layout:section_cta_banner}
            <div style="background: #0CAFEB; padding: 1rem; margin-bottom: 1rem;">
            {fields}
                {ff_webpage_layout:pf_heading}
                    <h1>{content}</h1>
                {/ff_webpage_layout:pf_heading}
                {ff_webpage_layout:pf_intro_text}
                    <p>{content}</p>
                {/ff_webpage_layout:pf_intro_text}
            {/fields}
            </div>
        {/ff_webpage_layout:section_cta_banner}
    {/ff_webpage_layout}
{/exp:channel:entries}
```

**Error Messages**
On the frontend,  the Hero section it renders like this:

<img width="1235" alt="Screenshot 2024-07-10 at 16 49 17" src="https://github.com/ExpressionEngine/Coilpack/assets/2130276/87d1ee00-bdc8-439d-971f-36bb468789ae">

In the entry preview the hero section renders like this:

<img width="1524" alt="Screenshot 2024-07-10 at 16 50 58" src="https://github.com/ExpressionEngine/Coilpack/assets/2130276/86ce9552-6b9f-4fe2-a7d5-3632b40b57ab">

I had thought that there was something wrong with my markup, but when I inspect the page I can see EE is looping my whole div, 1 per field.

**Screenshots / Videos / Template Code**

**Environment Details:**
 - Version: 7.4.11
 -  Using structure for managing layouts

**Comment by bryannielsen** [comment]
@acutedeveloper thanks for reporting this and helping me get setup to replicate it!  I can confirm the issue is in 7.4.11 and that it is fixed by this pull request #4030 which is scheduled to release with version 7.5.0

I'll leave this issue open until the fix is released

**Comment by bryannielsen** [comment]
Resolved in [7.5.0](https://github.com/ExpressionEngine/ExpressionEngine/releases/tag/7.5.0)
